iotfunctions/metadata.py

```diff
@@ -228,7 +228,10 @@ def __init__(self, name, db, *args, **kwargs):
         except AttributeError:
             self.logical_name = name
 
-        if db.db_type == 'db2':
+
+        if db == None:
```

Comment:
@sedgewickmm18 I thought we introduced this temporarily for testing without db.

Reply:
@sedgewickmm18 I saw your comment 
https://github.com/ibm-watson-iot/functions/pull/126#issuecomment-578214906